hack/make-rules/build-images.sh

```diff
@@ -34,7 +34,7 @@ platform_array=("${arch_array[@]/#/linux/}")
 PLATFORMS=$(IFS=","; echo "${platform_array[*]}")
 REGISTRY=${REGISTRY:-"ghcr.io/kubewharf"}
 OUTPUT_TYPE="docker"
```

Comment:
```suggestion
OUTPUT_TYPE=${OUTPUT_TYPE:-"docker"}
```
The value of `OUTPUT_TYPE` here is fixed as the `docker` type. If we need to support multiple types of `OUTPUT_TYPE`, please change it to receive a different variable value.